napari/_qt/qt_viewer.py

```diff
@@ -152,8 +153,12 @@ def __init__(self, viewer):
         self.dockLayerList.setMaximumWidth(258)
         self.dockLayerList.setMinimumWidth(258)
 
+        # Only created if using perfmon.
         self.dockPerformance = self._create_performance_dock_widget()
 
+        # Only created if using async rendering.
+        self.dockRender = self._create_render_dock_widget()
+
```

Comment:
Can we make this off by default, something like

```suggestion
    if self.dockRender is not None:
        self.dockRender.setVisible(False)
```

Right now when I open my pathology example I just get an empty widget at the right. It's pretty easy to turn on with the shortcut

Reply:
See latest commit, the render widget now only appears with octree visuals. It does nothing with non-octree visuals anyway so this is better.

Reply:
nice, thanks!